test/dialect/postgresql/test_types.py

```diff
@@ -8,6 +8,8 @@
 import re
 import uuid
 
+from asyncpg import BitString
```

Comment:
the tests may be run without asyncpg, so this should be in try/catch or inside a method of the test class that uses it

Reply:
Ok, I added the the try/catch again. Thanks!